sharding/testhelpers/unit_test_suite.go

```diff
@@ -71,6 +71,8 @@ func (t *ShardingUnitTestSuite) SetupTest() {
 	t.setupShardingFerry()
 	t.dropTestDbs()
 
+	testhelpers.SetupTest()
```

Comment:
I added this so we can get some debug log, which proved to be useful when I was trying to debug this implementation. I could remove it, tho.